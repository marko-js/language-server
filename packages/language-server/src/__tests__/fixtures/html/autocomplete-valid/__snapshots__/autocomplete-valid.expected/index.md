## Diagnostics
### Ln 2, Col 2
```marko
  1 | <label for="input">Name:</label>
> 2 | <input type="text" id="input" name="name" autocomplete="invalid-value">
    |  ^^^^^ Fix all of the following:
  the autocomplete attribute is incorrectly formatted
```

