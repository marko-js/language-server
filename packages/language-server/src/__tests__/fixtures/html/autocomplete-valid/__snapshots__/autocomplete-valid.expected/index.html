<label data-marko-node-id="0" for="input">Name:</label><input data-marko-node-id="1" type="text" id="input" name="name" autocomplete="invalid-value">